zipline/data/minute_bars.py

```diff
@@ -465,6 +489,17 @@ def __init__(self,
     def first_trading_day(self):
         return self._start_session
 
+    def ohlc_ratio_for_sid(self, sid):
+        if self._ohlc_ratios_per_sid is not None:
+            try:
+                return self._ohlc_ratios_per_sid[sid]
+            except KeyError:
```

Comment:
I opted here to share the return below for both the case where the sid is not present in the dict, and when no dict was passed to begin with. Do you think separate returns would be better?